config/kubernetes/sig-docs/teams.yaml

```diff
@@ -1,151 +1,170 @@
 teams:
-  sig-docs-en-owners:
-    description: ""
+  sig-docs-en-owners: 
+    description: English content admins 
+    maintainers:
+    - bradamant3
```

Comment:
All users need to defined as `members` instead of `maintainers`. Ref: #563 

I have a PR to add a test for it https://github.com/kubernetes/org/pull/592, so this should be caught earlier in the future. :grimacing: